src/apps/reservation-list/list/reservation-list.tsx

```diff
@@ -256,27 +256,30 @@ const ReservationList: FC<ReservationListProps> = ({ pageSize }) => {
 
   return (
     <>
-      {modalIds.length === 0 && (
-        <div className="reservation-list-page">
-          <h1 className="text-header-h1 m-32">
-            {t("reservationListHeaderText")}
-          </h1>
-          {user && <ReservationPauseToggler user={user} />}
-          {allListsEmpty && <EmptyReservations />}
-          {!allListsEmpty && (
-            <DisplayedReservations
-              openReservationDetailsModal={openReservationDetailsModal}
-              readyForPickupReservationsFBS={readyForPickupReservationsFBS}
-              readyForPickupReservationsPublizon={
-                readyForPickupReservationsPublizon
-              }
-              reservedReservationsFBS={reservedReservationsFBS}
-              reservedReservationsPublizon={reservedReservationsPublizon}
-              pageSize={pageSize}
-            />
-          )}
-        </div>
-      )}
+      <div
+        className="reservation-list-page"
+        style={
+          modalIds.length > 0 ? { overflow: "hidden", height: "100vh" } : {}
```

Comment:
Wouldn't it be prettier to flip a class instead of hardcoding the style?
And since you are doing the same [here](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/311/files#diff-2cdd62020bea6d73b0ed5c5b7ead29e571bdb06dedcfeb649598dcc202fe42c3R162) I think you should create a function for it.